bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/wrappers/veneer/BigtableHBaseVeneerSettings.java

```diff
@@ -15,12 +15,14 @@
  */
 package com.google.cloud.bigtable.hbase.wrappers.veneer;
 
+import static com.google.cloud.bigtable.config.BulkOptions.BIGTABLE_BULK_THROTTLE_TARGET_MS_DEFAULT;
```

Comment:
can you also add a small test asserting the default is used when not specified?

Reply:
+1

Reply:
done